scss/custom/_application.scss

```diff
@@ -0,0 +1,55 @@
+.application {
+  color: $application-color;
+  background: $application-bg;
+  padding: 0 $application-padding-x;
+  flex: 0 0 auto;
+
+  .btn-primary {
+    white-space: normal;
+  }
+
+  &-header {
+    background: $brand-primary;
+    padding: 1.5rem;
+    position: relative;
+
+    .h1 {
+      color: $white;
+    }
+
+    .application-title {
+      font-size: 1.1rem;
+    }
+
+    &:after {
+      content: "";
+      width: 0;
+      height: 0;
+      border-left: .5rem solid transparent;
+      border-right: .5rem solid transparent;
+      border-top: .5rem solid $brand-primary;
+      position: absolute;
+      top: auto;
+      bottom: -.49rem;
```

Comment:
Do you have to substract .01 because of a small "border"? Did you try top: 100% instead?